.github/workflows/docker.yml

```diff
@@ -108,6 +110,25 @@ jobs:
       - name: 'Register gcloud as Docker credential helper'
         run: |
             gcloud auth configure-docker -q
+      - name: "Configure AWS credentials"
+        uses: aws-actions/configure-aws-credentials@v1-node16
+        with:
+          role-to-assume: arn:aws:iam::308190735829:role/gh-dockerhub-releaser
+          aws-region: eu-west-2
+        if: github.event_name == 'release'
+      - name: Read secrets from AWS Secrets Manager into environment variables
+        uses: aws-actions/aws-secretsmanager-get-secrets@v1
+        with:
+          secret-ids: |
+            DOCKERHUB, github-actions/rdxworks/dockerhub-images/release-credentials
+          parse-json-secrets: true
+        if: github.event_name == 'release'
+      - name: Login to Docker Hub
+        uses: docker/login-action@v2
+        with:
+          username: ${{env.DOCKERHUB_USERNAME}}
+          password: ${{env.DOCKERHUB_TOKEN}}
+        if: github.event_name == 'release'
```

Comment:
looks good. Although we need to define condition in multiple places, still better alternative.